api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirGlobalPropertyDaoImpl.java

```diff
@@ -47,10 +46,16 @@ public GlobalProperty getGlobalPropertyObject(String property) {
 	}
 	
 	@Override
-	public List<String> getGlobalProperties(String... properties) {
+	public HashMap<String, String> getGlobalProperties(String... properties) {
```

Comment:
I'd still have this return a `Map<String, String>` rather than `HashMap<String, String>`